x-pack/test/alerting_api_integration/spaces_only/tests/actions/get_all.ts

```diff
@@ -54,6 +54,9 @@ export default function getAllConnectorsTests({ getService }: FtrProviderContext
           is_preconfigured: true,
           is_deprecated: false,
           is_system_action: false,
+          config: {
```

Comment:
are these test changes correct? I don't see any of the preconfigured test connectors adding the `exposeConfig` option.